src/components/seo.tsx

```diff
@@ -78,7 +81,7 @@ export const PureSEO = ({
   )
 }
 
-const SEO = ({ description, lang, meta, title }: SEOProps) => {
+const SEO = ({ description }: SEOProps): ReactElement | null => {
```

Comment:
```suggestion
const SEO: React.FC<SEOProps> = ({ description }) => {
```
Navrhoval bych použít jiný přístup

Reply:
@fidelman toto pak vyhazuje error ` react/prop-types: 'description' is missing in props validation`.... Snažil jsem se najít nějaký ESlint plugin, ale neúspěšně. Navrhuješ tedy toto pravidlo zakázat?

Reply:
> @fidelman toto pak vyhazuje error ` react/prop-types: 'description' is missing in props validation`.... Snažil jsem se najít nějaký ESlint plugin, ale neúspěšně. Navrhuješ tedy toto pravidlo zakázat?

🤦‍♂️ jo vidim https://github.com/yannickcr/eslint-plugin-react/issues/2654, to bylo mozne v tslint.
a pak navrhuji
```suggestion
const SEO = ({ description }: SEOProps): JSX.Element => {
```

Reply:
Ani není nutné return value explicitně typovat typescript to pochopí.
<img width="492" alt="Screenshot 2020-10-20 at 00 43 37" src="https://user-images.githubusercontent.com/3129012/96519458-544d0100-126d-11eb-83b3-b0924337146a.png">

Reply:
> Ani není nutné return value explicitně typovat typescript to pochopí.
> <img alt="Screenshot 2020-10-20 at 00 43 37" width="492" src="https://user-images.githubusercontent.com/3129012/96519458-544d0100-126d-11eb-83b3-b0924337146a.png">

🤷‍♀️
<img width="1151" alt="Screenshot 2020-10-20 at 21 33 23" src="https://user-images.githubusercontent.com/15198653/96635247-e4e21a80-131b-11eb-877f-904fe90740de.png">